src/main/java/eu/dissco/core/translator/maven/MavenRunner.java

```diff
@@ -0,0 +1,63 @@
+package eu.dissco.core.translator.maven;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MavenRunner {
```

Comment:
Did you write this or is this from another source?

Reply:
Both, tried to make github Copilot do some work, you know to cover its fee.